src/content/h3/guides/shaders/pbr-shader/readme.md

```diff
@@ -15,6 +15,9 @@ Physically based rendering (PBR) is a way of rendering images that mimics how li
 
 With the release of the weapon and vehicle skins update to Halo 3 MCC, 343 Industries implemented a new material_model called **cook_torrance_pbr_maps**. This allowed for the menu skins that were authored in PBR to be replicated in-engine when enabled. While not 100% true PBR, this shader allows artists to now have a more modern rendering technique in Halo 3.
 
+
+**NOTE:** *There is no definitive way to utilize this shader at this time, as even 343 Industries uses inconsistent shader settings with this shader. This guide however is a good baseline setup to start with.*
+
```

Comment:
Please use the `.alert` syntax for this (see other pages for an example of how to do it.